<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>651</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit_DispMsg">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>431</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Connect">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>连接服务器</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Disconnect">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>断开服务器</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_InputMsg">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Send">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>150</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>发送信息</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxIp">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxPort">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>3666</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelip">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>服务器地址：</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelport">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>服务器端口：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonFile">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>选择文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSend">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>发送文件</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
